src/src/tx_main.cpp

```diff
@@ -666,7 +682,11 @@ static void setupTarget()
 
 void setup()
 {
+#ifdef TARGET_AXIS_THOR_2400_TX
+  Serial.begin(420000);
```

Comment:
Maybe it is better to have this one as define?

Reply:
We have a similar define on the receiver side: RCVR_UART_BAUD

Reply:
on commit 45fefe8, use define TX_UART_BAUD and axis target override it